rllib/agents/trainer.py

```diff
@@ -976,6 +996,31 @@ def _export_model(self, export_formats, export_dir):
             exported[ExportFormat.MODEL] = path
         return exported
 
+    def import_model(self, import_file):
```

Comment:
Should we also have an export_model to H5 (for symmetry?)

Reply:
Let's do this in another comprehensive PR that covers all these export/import API methods for Trainers/Policies/Models. I feel like more thought and discussion has to go into all this.

Reply:
Sounds good. Could you look over the other comments and see if they should be addressed here or later on/

Reply:
Yes, will do. ...